plugins/poll/config/locales/server.en.yml

```diff
@@ -4,6 +4,7 @@ en:
     poll_maximum_options: "Maximum number of options allowed in a poll."
     poll_edit_window_mins: "Number of minutes after post creation during which polls can be edited."
     poll_minimum_trust_level_to_create: "Define the minimum trust level needed to create polls."
+    poll_create_allowed_groups: "The groups that are allowed to create polls."
```

Comment:
Do we need to add the entry under `keywords:`?

Reply:
Ah yeah good point, will do